### PR TITLE
Introducing error code 501 "Not Implemented"

### PR DESCRIPTION
**Pull request by dfischer-tech**
This PR is related to #81 and align to PR #104 reformatting errors according to the guidelines.

### CODE REVIEW

**Review by jlurien — APPROVED** [review]

**Comment by hdamker** [comment]
Thanks @jlurien for the first review. Guess this PR is an easy one. Waiting for one additional review/approval.

**Review by shilpa-padgaonkar — APPROVED** [review]

File: code/API_definitions/qod-api.yaml
Comment (dfischer-tech) [reply code/API_definitions/qod-api.yaml]: Clearly agreed.

**Pull request merged**
